app/views/vm_common/console_vmrc.html.haml

```diff
@@ -367,36 +367,42 @@
         // Start up VMRC
         vmrc = document.getElementById("vmrc");
 
-        vmNameTooltip = $("#vmNameTooltip");
-        var vmTitle = $("#vmTitle");
+        if (typeof vmrc.isReadyToStart === 'undefined') {
+          window.location.href = "vmrc://clone:#{ticket}@#{host}:443/?moid=#{vmid}";
```

Comment:
@Fryguy maybe [this](http://ruby-doc.org/stdlib-2.0.0/libdoc/uri/rdoc/URI/HTTP.html), but i'm not sure about the IPv6. Will try out...